docker/dev-ubuntu/Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM ubuntu:bionic
+#Use Official Docker Images for OCAML/OPAM
+#https://github.com/ocaml/infrastructure/wiki/Containers
+FROM ocaml/opam2:debian-stable
```

Comment:
This just means that we're not testing our install_ocaml script anymore, but that's fine with me. @seantalts , are you also OK with that?

Reply:
I've done this so we use the latest images build by the official team. This means a stable image, always up to date and reviewed by the devs of ocaml. Besides this, for us it means faster build/test times since we're not spending time building it from source, just using an already pre-build image.

Reply:
Makes sense! Naive question [I know little to nothing about Jenkins/Docker]: in the new setup, how often would all the library dependencies have to be installed by the build/test machine? Are the installations of those being cached or would they be installed every time we run tests?

Reply:
They're in cache :D

Reply:
Great!

Reply:
Oh yeah woops, we need to make sure the scripts are under test in at least one of these test runs. Can you revert it back to using the `install_ocaml` script?